tests/utils/renderCfg.json

```diff
@@ -0,0 +1,13 @@
+{
+    "renderCfg": [{
+        "package": ".*",
```

Comment:
I would have expected the packages listed in reverse order,  from  specific ones like "package", to wildcards like ".*".

The match would ten be "1st one wins" and if no matches are found the generic ".*" is taken.